cypress/integration/full_run_with_amounts.spec.js

```diff
@@ -320,7 +258,7 @@ describe('Full run through saying "yes" to everything', function() {
 
     //check some table data
     //until we have a more firm grasp on how we're shaping the total refund, i'm just checking benefits
-    checkTableRows(cy, getBenefitsBreakdownRows(this.user))
```

Comment:
Is it still necessary to have two versions of the table / two util functions here?

Reply:
Good point! I updated removing the non-lite version of checkBenefitsRows, and I modified checkTableRows to accept a string for css class name, so that we don't need a lite/non-lite version